src/components/Startups/index.tsx

```diff
@@ -0,0 +1,131 @@
+            <section className="text-center py-40 max-w-screen-lg mx-auto relative px-5">
+                <div className="absolute right-0 md:bottom-0 -bottom-12 md:max-w-[240px] max-w-[150px]">
+                    <StaticImage width={240} src="./images/belay-on.png" />
+                </div>
+                <div className="absolute left-0 bottom-0 max-w-[120px] md:max-w-[200px]">
+                    <StaticImage width={200} src="./images/on-belay.png" />
+                </div>
+                <div className="relative">
+                    <h1>Let's do this together</h1>
+                    <p>PostHog can help your startup get to where it needs to be</p>
+                </div>
+            </section>
+            <section className="grid md:grid-cols-2 gap-y-8 md:gap-y-0 md:gap-x-12 max-w-[1100px] mx-auto px-5 my-16">
+                <div>
+                    <h4>Benefits with PostHog for startups</h4>
+                    <ul className="list-none p-0 m-0 grid grid-flow-row md:grid-cols-2 gap-y-4">
+                        {benefits.map((benefit) => {
+                            return (
+                                <li className="flex items-start space-x-2" key={benefit}>
+                                    <Check2 className="w-4 opacity-57 flex-shrink-0 mt-[2px]" />
+                                    <span className="leading-tight">{benefit}</span>
+                                </li>
+                            )
+                        })}
+                    </ul>
+                    <div className="mt-6 md:mt-8">
+                        <Spotlight {...spotlight} />
+                    </div>
+                </div>
+                <div className="flex-shrink-0">
+                    <h3>Apply for the PostHog for startups program</h3>
+                    <HubSpotForm
+                        validationSchema={validationSchema}
+                        formID="aa91765b-e790-4e90-847e-46c7ebf43705"
+                        customMessage={
+                            <>
+                                <h4>
+                                    ✅ <strong>Message received!</strong>
+                                </h4>
+                                <p>
+                                    A member of the PostHog team will get back to you as soon as we've had a chance to
+                                    review your information.&nbsp;
+                                </p>
+                                <p className="mb-0">
+                                    In the meantime, why not join <Link to="/slack">our Slack community</Link>?
+                                </p>
+                            </>
+                        }
+                        customFields={{
+                            self_registration_raised: {
+                                type: 'radioGroup',
+                                options: [
+                                    { label: 'Boostrapped', value: -1 },
```

Comment:
Is `0` a value we can put in here? Looking at how the breakdowns work, `0` technically would represent _no money raised_ aka bootstrapped. (Don't know why I didn't think of this before!